fuzz/fuzz_targets/fast-forward.rs

```diff
@@ -57,27 +57,25 @@ fuzz_target!(|data: &[u8]| {
     let new_solution = node_to_bytes(&a, new_solution).expect("serialize new solution");
 
     // run original spend
-    let conditions1 = run_puzzle(
+    let conditions1 = run_puzzle::<MempoolVisitor>(
         &mut a,
         spend.puzzle_reveal.as_slice(),
         spend.solution.as_slice(),
         &spend.coin.parent_coin_info,
         spend.coin.amount,
         11000000000,
         0,
-        &mut MempoolPolicy::default(),
     );
 
     // run new spend
-    let conditions2 = run_puzzle(
+    let conditions2 = run_puzzle::<MempoolVisitor>(
         &mut a,
         spend.puzzle_reveal.as_slice(),
         new_solution.as_slice(),
         &new_coin.parent_coin_info,
         new_coin.amount,
         11000000000,
         0,
-        &mut MempoolPolicy::default(),
```

Comment:
I think you have to leave this line in. Otherwise we can't ever add a callback that has a context, like a mempool sniffer would almost certainly need.